docs/components/tree-list.md

```diff
@@ -0,0 +1,330 @@
+# Tree List
+
+**KTreeList** - A drag-n-drop reorderable list component.
+
+::: tip
+It can be tricky to determine if an item will be places below an item as a sibling or as a child when dropping. Items to be dropped as children should be dropped near the bottom border line of the parent. If this will be the first child of an item, you will notice the margin between the two items disappearing if you are hovering in the right place.
```

Comment:
For this problem, I think increasing the drop zone height could give more space to get the feeling that the element will be placed there?

Reply:
If I update that it adds more margin between the items making it stop matching the design.

Reply:
10px might be a big much, we have 4px right now. Here is how it would look with 10px.
![image](https://user-images.githubusercontent.com/67973710/203652401-32523856-625e-433c-8bbf-eca878032eeb.png)